doc/apiv3-documentation.apib

```diff
@@ -2060,7 +2060,7 @@ Revisions are sets of updates to files in the context of repositories linked in
 ## Local Properties
 | Property                | Description                                                                                                                                                 | Type        | Constraints | Supported operations |
 |:-----------------------:| ------------------------------------------------------------------------------------------------------------------------------------------------------------| ----------- | ----------- | -------------------- |
-| id                      | Revisions's id, assigned by OpenProject                                                                                                                     | Integer     | x > 0       | READ                 |
+| id                      | Revision's id, assigned by OpenProject                                                                                                                     | Integer     | x > 0       | READ                 |
```

Comment:
Yeah, that is totally GitHub messing things up... just hit F5 (aka *the reload button*) and the problem will be gone...